aea/cli/create.py

```diff
@@ -126,7 +127,7 @@ def create_aea(
 
     if author is not None:
         if is_local:
-            do_init(author, False, False, False)  # pragma: nocover
```

Comment:
Why do we set http as default?

Reply:
I was not sure which one to use. Should I use IPFS as default?

Reply:
Ideally yes, but it might break all sorts of tests. So I am ok for backwards compatibility to set it as is for now. Then the question is, how do I update it to point to ipfs via cli?

Another approach, during init ask for registry backend selection and set to default to http.

Reply:
I've already added functionality for this https://github.com/valory-xyz/open-aea/blob/69e389f06b258be4fa8a2ce688bff5ce6592e09c/aea/cli/init.py#L38